pyro/distributions/transforms/__init__.py

```diff
@@ -5,45 +5,36 @@
-                                                           conditional_affine_coupling)
-from pyro.distributions.transforms.basic import ELUTransform, LeakyReLUTransform, elu, leaky_relu
-from pyro.distributions.transforms.batchnorm import BatchNorm, batchnorm
-from pyro.distributions.transforms.block_autoregressive import BlockAutoregressive, block_autoregressive
-from pyro.distributions.transforms.cholesky import CorrLCholeskyTransform
-from pyro.distributions.transforms.discrete_cosine import DiscreteCosineTransform
-from pyro.distributions.transforms.generalized_channel_permute import (ConditionalGeneralizedChannelPermute,
-                                                                       GeneralizedChannelPermute,
-                                                                       conditional_generalized_channel_permute,
-                                                                       generalized_channel_permute)
-from pyro.distributions.transforms.haar import HaarTransform
-from pyro.distributions.transforms.householder import (ConditionalHouseholder, Householder, conditional_householder,
-                                                       householder)
-from pyro.distributions.transforms.lower_cholesky_affine import LowerCholeskyAffine
-from pyro.distributions.transforms.matrix_exponential import (ConditionalMatrixExponential, MatrixExponential,
-                                                              conditional_matrix_exponential, matrix_exponential)
-from pyro.distributions.transforms.neural_autoregressive import (ConditionalNeuralAutoregressive, NeuralAutoregressive,
-                                                                 conditional_neural_autoregressive,
-                                                                 neural_autoregressive)
-from pyro.distributions.transforms.ordered import OrderedTransform
-from pyro.distributions.transforms.permute import Permute, permute
-from pyro.distributions.transforms.planar import ConditionalPlanar, Planar, conditional_planar, planar
-from pyro.distributions.transforms.polynomial import Polynomial, polynomial
-from pyro.distributions.transforms.radial import ConditionalRadial, Radial, conditional_radial, radial
-from pyro.distributions.transforms.spline import ConditionalSpline, Spline, conditional_spline, spline
-from pyro.distributions.transforms.spline_autoregressive import (ConditionalSplineAutoregressive, SplineAutoregressive,
-                                                                 conditional_spline_autoregressive,
-                                                                 spline_autoregressive)
-from pyro.distributions.transforms.spline_coupling import SplineCoupling, spline_coupling
-from pyro.distributions.transforms.sylvester import Sylvester, sylvester
+from ..constraints import IndependentConstraint, corr_cholesky_constraint, ordered_vector, sphere
```

Comment:
This is just (1) replace `pyro.distributions.transforms` with `.`; then (2) run `isort`.